docker/bin/zookeeperTeardown.sh

```diff
@@ -40,7 +47,8 @@ MYID=`cat $MYID_FILE`
 ZNODE_PATH="/zookeeper-operator/$CLUSTER_NAME"
 CLUSTERSIZE=`java -Dlog4j.configuration=file:"$LOG4J_CONF" -jar /root/zu.jar sync $ZKURL $ZNODE_PATH`
 echo "CLUSTER_SIZE=$CLUSTERSIZE, MyId=$MYID"
-if [[ -n "$CLUSTERSIZE" && "$CLUSTERSIZE" -lt "$MYID" ]]; then
+CLUSTERSIZE=$(($CLUSTERSIZE+$OFFSET-1))
+if [[ -n "$CLUSTERSIZE" && "$(($CLUSTERSIZE+$OFFSET-1))" -lt "$MYID" ]]; then
```

Comment:
I see here you compute the `CLUSTERSIZE` twice?
shouldn' this be ?
```suggestion
if [[ -n "$CLUSTERSIZE" && "$CLUSTERSIZE" -lt "$MYID" ]]; then
```

Reply:
right, that was a commit mistake, I will remove the `CLUSTERSIZE=$(($CLUSTERSIZE+$OFFSET-1))` like and let it compute it in the if statement.